server/web/helpers/jsonify.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = (context) => {//a helper used to format the object differently
+
+  return JSON.stringify(context);
+};
```

Comment:
remove this file! there already exists a file called josn,js inside helper dirctory which does the same thing. remove this file and replace jsonify with json in roles.handlebars.